model/config.go

```diff
@@ -113,6 +113,15 @@ type PrivacySettings struct {
 	ShowFullName     bool
 }
 
+type LegalAndSupportSettings struct {
+	TermsOfServiceLink string
+	PrivacyPolicyLink  string
+	AboutLink          string
+	HelpLink           string
+	ReportAProblemLink string
+	SupportEmail       string
```

Comment:
The spec/associated help text specifically mentions it only changing the email given during the tutorial.  It makes sense if we want to consolidate it down to one email, but if we want them to be separate (e.g. support handles emails sent from users needing help, whereas feedback is the email things like auto-generated notification emails are sent from) we will need to keep two separate settings.  Once again fine either way, but should probably have a discussion.